<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="11">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QLineEdit" name="fileEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="fileButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="VolumeLayout">
      <item>
       <widget class="QSlider" name="volumeSlider">
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="Filtro31Hz">
      <item>
       <widget class="QSlider" name="filtro31HzSlider">
        <property name="minimum">
         <number>-25</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QHBoxLayout" name="Filtro63Hz">
      <item>
       <widget class="QSlider" name="filtro63HzSlider">
        <property name="minimum">
         <number>-25</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <layout class="QHBoxLayout" name="Filtro125Hz">
      <item>
       <widget class="QSlider" name="filtro125HzSlider">
        <property name="minimum">
         <number>-25</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="4">
     <layout class="QHBoxLayout" name="Filtro250Hz">
      <item>
       <widget class="QSlider" name="filtro250HzSlider">
        <property name="minimum">
         <number>-25</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="5">
     <layout class="QHBoxLayout" name="Filtro500Hz">
      <item>
       <widget class="QSlider" name="filtro500HzSlider">
        <property name="minimum">
         <number>-25</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="6">
     <layout class="QHBoxLayout" name="Filtro1kHz">
      <item>
       <widget class="QSlider" name="filtro1kHzSlider">
        <property name="minimum">
         <number>-25</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="7">
     <layout class="QHBoxLayout" name="Filtro2kHz">
      <item>
       <widget class="QSlider" name="filtro2kHzSlider">
        <property name="minimum">
         <number>-25</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="8">
     <layout class="QHBoxLayout" name="Filtro4kHz">
      <item>
       <widget class="QSlider" name="filtro4kHzSlider">
        <property name="minimum">
         <number>-25</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="9">
     <layout class="QHBoxLayout" name="Filtro8kHz">
      <item>
       <widget class="QSlider" name="filtro8kHzSlider">
        <property name="minimum">
         <number>-25</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="10">
     <layout class="QHBoxLayout" name="Filtro16kHz">
      <item>
       <widget class="QSlider" name="filtro16kHzSlider">
        <property name="minimum">
         <number>-25</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="volume">
      <property name="text">
       <string>Volumen</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="filtro31HzLabel">
      <property name="text">
       <string>31 Hz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="filtro63HzLabel">
      <property name="text">
       <string>63 Hz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="filtro125HzLabel">
      <property name="text">
       <string>125 Hz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="filtro250HzLabel">
      <property name="text">
       <string>250 Hz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="filtro500HzLabel">
      <property name="text">
       <string>500 Hz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLabel" name="filtro1kHzLabel">
      <property name="text">
       <string>1 kHz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QLabel" name="filtro2kHzLabel">
      <property name="text">
       <string>2 kHz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QLabel" name="filtro4kHzLabel">
      <property name="text">
       <string>4 kHz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QLabel" name="filtro8kHzLabel">
      <property name="text">
       <string>8 kHz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="10">
     <widget class="QLabel" name="filtro16kHzLabel">
      <property name="text">
       <string>16 kHz</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
